torch/csrc/jit/codegen/fuser/cpu/fused_kernel.cpp

```diff
@@ -196,11 +196,18 @@ struct CompilerConfig {
 #ifdef _MSC_VER
   std::string cxx = "cl";
   const std::string openmp_flags = "/openmp";
+#elif defined(__APPLE__)
+  std::string cxx = "clang++";
+  const std::string openmp_flags = "";
 #else
   std::string cxx = "g++";
   const std::string openmp_flags = "-fopenmp";
 #endif
+#if defined(__APPLE__)
+  bool openmp = false;
```

Comment:
can we add a comment for posterity on why we're doing this?

Reply:
Sure, OpenMP support is MacOS devtools is explicitly disabled:
```
   $ clang -c omp.c -fopenmp
   clang: error: unsupported option '-fopenmp'
```

Reply:
Updated the PR to rely less on the platform, but more on the compilation flags (i.e. openmp is set to false if _OPENMP flag is not defined)